.github/workflows/very-expensive-test.yml

```diff
@@ -0,0 +1,43 @@
+name: Very Expensive Test
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      # WARN: This will run very expensive test every time ANY label is added
+      - labeled
+  schedule:
+    - cron: '0 0 * * *' # Runs nightly at 0AM UTC
```

Comment:
any idea what it's going to cost to run this daily? given the current very expensive and how long they take to run on the current hardware, are we looking at cents or dollars here?

Reply:
I'd say about $1 per very expensive test run.

Reply:
Ok, sounds good.  Let go with it for now and we can adjust.  It definitely helps to find out soon after a bad commit if we broke something...